app/assets/stylesheets/admin/openfoodnetwork.css.scss

```diff
@@ -2,7 +2,7 @@
 
 input[type="submit"], input[type="button"], button, .button {
   cursor: pointer;
-  padding: 6.5px 15px;
+  font-size: 85%;
```

Comment:
this is changing all buttons in the backoffice, probably worth mentioning it in the test section of the PR.

Reply:
It's partially reverting the last change I made to try to fix the visual regression on the buttons, but this time it's better. I compared the CSS with v1 and found what was missing. I mentioned it in the test section. :+1: